.github/workflows/ci.yml

```diff
@@ -150,12 +150,46 @@ jobs:
                 echo "javac"
             fi`"
 
+      
+    
+      - name: Get java version
+        id: java-version
+        run: |
+           echo '::set-output name=JAVA_VERSION::
+           cat java_version.txt'
+
+      - name: Set up JDK 8
+        if: steps.java-version.outputs.JAVA_VERSION == '8'
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
       - name: Set up JDK 11
+        if: steps.java-version.outputs.JAVA_VERSION == '11'
         uses: actions/setup-java@v2
         with:
           java-version: '11'
           distribution: 'adopt'
 
+
+
+
+
+      # - name: Set up JDK 
+      #   # if: steps.java-version.outputs.JAVA_VERSION == '11'
+      #   uses: actions/setup-java@v1
+      #   with:
+      #     java-version: ${{ matrix.java-version }}
+      #     # distribution: 'adopt'
+
+      #  - name: Set up JDK 13
+      #   if: steps.java-version.outputs.JAVA_VERSION == '13'
+      #   uses: actions/setup-java@v2
+      #   with:
+      #     java-version: '13'
+      #     distribution: 'adopt'
+
```

Comment:
We don't need this line as of now.

Reply:
yes mukesh i was trying with matrix so set up jdk for that but we can keep java 13